multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -123,7 +123,11 @@ type ClusterSetStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 
-// ClusterSet is the Schema for the clustersets API.
+// +kubebuilder:printcolumn:name="Leader Cluster Namespace",type=string,JSONPath=`.spec.namespace`,description="The leader cluster Namespace for the ClusterSet"
+// +kubebuilder:printcolumn:name="Total Clusters",type=string,JSONPath=`.status.totalClusters`,description="Total number of clusters in the ClusterSet"
+// +kubebuilder:printcolumn:name="Ready Clusters",type=string,JSONPath=`.status.readyClusters`,description="Number of ready clusters in the ClusterSet"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="Ready Member Clusters",type=string,JSONPath=`.status.readyClusters`,description="Number of ready member clusters in the ClusterSet"
```
Leader cluster is not included in the ready clusters, so better to add `Member` here. Otherwise user might be confused to see inconsistent data between `totalClusters` and `readyClusters`

Reply:
I feel we should change totalClusters to include only member clusters, as we support only a single leader cluster. What you think?

Reply:
Sounds reasonable, I can update totalClusters and GatewayIPPrecedence after you merge this.